lib/log.js

```diff
@@ -208,46 +225,124 @@ const util = require('util'),
+                existLogFile = 0;
+
+            async.series([
+                _makeSession,
+                _setparam,
+                function(next) {
+                    options.session.run(cmd, process.stdin, _checkService, process.stderr, function(err) {
+                        if (err) {
+                            return next(err);
+                        }
+                        if (!logDaemonStatus) {
+                            return next(errHndl.getErrMsg("NOT_MATCHED_LOGDAEMON", options.currentDaemon === "journald" ? "pmlogd" : "journald"));
+                        }
+
+                        cmd = `ls -al ${logFilePath}`;
+                        options.session.run(cmd, process.stdin, _checkFile, process.stderr, function(error) {
+                            if (error) {
+                                return next(err);
+                            }
+                            if (!existLogFile) {
+                                return next(errHndl.getErrMsg("NOT_EXIST_LOGFILE"));
+                            }
+                            next();
+                        });
                     });
+                },
+            ], function(err) {
+                if (err) {
+                    console.log("CLI's current log daemon : " + options.currentDaemon);
+                    next(err, null);
+                } else {
+                    next(null, {msg : "CLI's current log daemon : " + options.currentDaemon
+                                        + "\nThe target's current log daemon : " + options.currentDaemon});
+                }
+            });
+
+            function _setparam(next) {
+                log.info("log#checkLogDaemon()#_setparam()");
+                if (options.currentDaemon === "journald") {
+                    serviceParam = "systemd-journald";
+                    logFilePath = "/run/log/journal";
+                } else if (options.currentDaemon === "pmlogd") {
+                    serviceParam = "pm-log-daemon";
+                    logFilePath = "/var/log/messages";
                 }
+
+                cmd = `systemctl status ${serviceParam}.service | tee`;
+                next();
             }
 
-            function _makeSession(next) {
-                makeSession(options, next);
+            function _checkService(data) {
+                log.info("log#checkLogDaemon()#_checkService()");
+                returnData += (Buffer.isBuffer(data))? data.toString() : data;
+
+                if (returnData.indexOf("active (running)") !== -1) {
+                    logDaemonStatus = 1;
+                }
             }
 
-            function _checkUser(next) {
-                checkUser(options, next);
+            function _checkFile(data) {
+                log.info("log#checkLogDaemon()#_checkFile()");
+                const str = (Buffer.isBuffer(data))? data.toString() : data;
+
+                if (str.length) {
+                    existLogFile = 1;
```

Comment:
Could you use "true/false" instead of "1"?

Reply:
done